e2e/next-sandbox/pages/zustand/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from "react";
 import create from "zustand";
 
-import { middleware } from "@liveblocks/zustand";
+import { liveblocks } from "@liveblocks/zustand";
```

Comment:
Moving to the Zustand ecosystem convention of naming the middlewares after your service/product/library. "Middleware" is too generic and every middle could be named that way. The `middleware` constant is still available as an export, but I've deprecated it in favor of `liveblocks`.